intro-HPC/examples/MATLAB/parpool.m

```diff
@@ -0,0 +1,2 @@
+c = parcluster('local')
+pool = parpool(c.NumWorkers)
```

Comment:
@redfast00 Did you check whether this returns the correct number of workers for partial jobs (e.g. with `ppn=4`)?

Reply:
Yes, I checked it with `ppn=5`, and it printed 5.